deploy/templates/consumption-azuredeploy.json

```diff
@@ -150,11 +150,11 @@
     "ResourceIdentityServiceType": {
```

Comment:
Can we rename this to ResourceIdentityResolutionType?  Will reflect current naming else where.

Reply:
Synced up offline. Would like to discuss the scope of this naming convention update, either only for the names on the templates or elsewhere for all other keys, variables etc. For either case, I'll do a separate PR.